app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java

```diff
@@ -314,7 +314,7 @@ public void onResume() {
         super.onResume();
         if (isVisible() || mIsInVRVideoMode) {
             mSession.setActive(true);
-            if (!SettingsStore.getInstance(getContext()).getLayersEnabled()) {
+            if (!SettingsStore.getInstance(getContext()).getLayersEnabled() && !mSession.hasDisplay()) {
```

Comment:
I wonder if we can just check if the mSession is active and then only call `setActive(true)` and `callSurfaceChanged()` if it isn't?

Reply:
I think is safer to keep this check. My worry is that other place of code calls SetActive before this method and in that case we would miss the surfaceChanged call